Godeps/Godeps.json

```diff
@@ -2363,82 +2363,82 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer",
-			"Comment": "v1.0.0-rc5-46-g871ba2e5",
+			"Comment": "v1.0.0-rc5-46-g871ba2e",
```

Comment:
we did not update this component, can you please change the SHA back to original? Looks like your environment chopped off the last digit

Reply:
@dims: I am not sure which part I did wrong, I followed the steps at https://github.com/kubernetes/community/blob/master/contributors/devel/godep.md which is pretty helpful.

1.  ./hack/run-in-gopath.sh godep restore
1.  ./hack/run-in-gopath.sh go get k8s.io/utils/exec
1.  ./hack/run-in-gopath.sh godep save

The stripping of last digt might be related with which part of my environment, git version?

Reply:
Yes, please update your git version.